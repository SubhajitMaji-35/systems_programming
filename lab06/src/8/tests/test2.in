8  7  6
4  3  2
0 -1 -2
-5 -4 -3
-1  0  1
3  4  5
