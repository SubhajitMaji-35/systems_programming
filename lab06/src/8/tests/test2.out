-29 -17 -5 
-8 -8 -8 
13 1 -11 
